eZ/Publish/Core/FieldType/Relation/Type.php

```diff
@@ -241,12 +241,14 @@ protected function getSortInfo(BaseValue $value)
      */
     public function fromHash($hash)
     {
-        $destinationContentId = $hash['destinationContentId'];
-        if ($destinationContentId !== null) {
-            $destinationContentId = (int)$destinationContentId;
+        if ($hash !== null) {
```

Comment:
In my opinion returning early would improve readability here.

Reply:
^

Reply:
Suggestion applied in https://github.com/ezsystems/ezplatform-kernel/pull/37/commits/e0e13406a98f7b72222c0987fa5e4ff006b65307